cluster/transport/service.go

```diff
@@ -38,20 +38,22 @@ type executor interface {
 }
 
 type Service struct {
-	members    members
-	executor   executor
-	address    string
-	ln         net.Listener
-	grpcServer *grpc.Server
-	log        *logrus.Logger
+	members     members
+	executor    executor
+	address     string
+	ln          net.Listener
+	grpcServer  *grpc.Server
+	grpcMaxSize int
+	log         *logrus.Logger
 }
 
-func New(ms members, ex executor, address string, l *logrus.Logger) *Service {
+func New(ms members, ex executor, address string, l *logrus.Logger, grpcMaxSize int) *Service {
```

Comment:
Same question as above, should I do something different here so the original function signature is preserved or is it fine as long I've replaced all usages in the weaviate codebase?

Reply:
i think it's fine , given the fact also it's just init func